stable/prometheus-blackbox-exporter/templates/daemonset.yaml

```diff
@@ -29,6 +29,7 @@ spec:
 {{ toYaml .Values.podAnnotations | indent 8 }}
         {{- end }}
     spec:
+      serviceAccountName: {{ template "prometheus-blackbox-exporter.serviceAccountName" . }}
```

Comment:
What if serviceAccount.create is false?

Reply:
it will use default as defined in master branch:
https://github.com/helm/charts/blob/master/stable/prometheus-blackbox-exporter/templates/_helpers.tpl#L36-L41